src/DssSpell.sol

```diff
@@ -118,28 +158,43 @@ contract DssSpellAction is DssAction {
             bidIncrease: 300, // 3%
             bidDuration: 6 hours,
             auctionDuration: 6 hours,
-            liquidationRatio: 14000 // 140%
+            liquidationRatio: 16500 // 165%
         });
-        addNewCollateral(UNIV2ETHUSDT_A);
-
-        // Faucet is currently set to zero address in Changelog.
-        //   We're cleaning it up this week and removing it from the list.
-        ChainlogAbstract(LOG).removeAddress("FAUCET");
-
-        // add UNIV2DAIUSDC to Changelog
-        setChangelogAddress("UNIV2DAIUSDC",             UNIV2DAIUSDC_GEM);
-        setChangelogAddress("MCD_JOIN_UNIV2DAIUSDC_A",  UNIV2DAIUSDC_JOIN);
-        setChangelogAddress("MCD_FLIP_UNIV2DAIUSDC_A",  UNIV2DAIUSDC_FLIP);
-        setChangelogAddress("PIP_UNIV2DAIUSDC",         UNIV2DAIUSDC_PIP);
-
-        // add UNIV2ETHUSDT to Changelog
-        setChangelogAddress("UNIV2ETHUSDT",             UNIV2ETHUSDT_GEM);
-        setChangelogAddress("MCD_JOIN_UNIV2ETHUSDT_A",  UNIV2ETHUSDT_JOIN);
-        setChangelogAddress("MCD_FLIP_UNIV2ETHUSDT_A",  UNIV2ETHUSDT_FLIP);
-        setChangelogAddress("PIP_UNIV2ETHUSDT",         UNIV2ETHUSDT_PIP);
+        addNewCollateral(UNIV2UNIETH_A);
+
+        // Rates changes
+        setIlkStabilityFee(ETH_A_ILK, FOUR_PT_FIVE_PCT, true);
+        setIlkStabilityFee(ETH_B_ILK, SEVEN_PT_FIVE_PCT, true);
+        setIlkStabilityFee(WBTC_ILK, FOUR_PT_FIVE_PCT, true);
+        setIlkStabilityFee(LINK_ILK, THREE_PT_FIVE_PCT, true);
+        setIlkStabilityFee(COMP_ILK, THREE_PCT, true);
+        setIlkStabilityFee(BAL_ILK, THREE_PT_FIVE_PCT, true);
+        setIlkStabilityFee(UNIV2DAIETH_ILK, TWO_PCT, true);
+        setIlkStabilityFee(UNIV2USDCETH_ILK, TWO_PT_FIVE_PCT, true);
+        setIlkStabilityFee(UNIV2WBTCETH_ILK, THREE_PT_FIVE_PCT, true);
+        setIlkStabilityFee(BAT_ILK, SIX_PCT, true);
+        setIlkStabilityFee(YFI_ILK, FIVE_PT_FIVE_PCT, true);
+
+        // Interim DAO Budget (Note: we are leaving daiJoin hoped from the Pause Proxy for future payments)
+        address MCD_JOIN_DAI = getChangelogAddress("MCD_JOIN_DAI");
+        VatAbstract(vat()).suck(vow(), address(this), 100_000 * RAD);
+        VatAbstract(vat()).hope(MCD_JOIN_DAI);
+        DaiJoinAbstract(MCD_JOIN_DAI).exit(0x73f09254a81e1F835Ee442d1b3262c1f1d7A13ff, 100_000 * WAD);
```

Comment:
I'm going to make a ticket to add this process to the exec-lib.

Reply:
Sounds good. I've already opened an issue on the repo: https://github.com/makerdao/dss-exec-lib/issues/48